apps/ledger-live-mobile/src/newArch/features/LandingPages/screens/LargeMoverLandingPage/index.tsx

```diff
@@ -1,16 +1,92 @@
-import { Flex, Text } from "@ledgerhq/native-ui";
-import React from "react";
+import { Flex } from "@ledgerhq/native-ui";
+import React, { useState } from "react";
+import { Card } from "./components/Card";
+import { CurrencyData, KeysPriceChange } from "@ledgerhq/live-common/market/utils/types";
+
+export const mockCurrencyData: CurrencyData = {
```

Comment:
can you extract that in `LargeMoverLandingPage/fixtures/currency.ts`

it will be easier when you remove everything you mocked